packages/rest-typings/src/v1/rooms.ts

```diff
@@ -423,6 +423,40 @@ const GETRoomsNameExistsSchema = {
 
 export const isGETRoomsNameExists = ajv.compile<GETRoomsNameExists>(GETRoomsNameExistsSchema);
 
+type RoomsIsUserMuted = { userId: string; roomId: string } | { username: string; roomId: string };
+
+const RoomsIsUserMutedSchema = {
+	type: 'object',
+	oneOf: [
+		{
+			properties: {
+				userId: {
+					type: 'string',
+				},
+				roomId: {
+					type: 'string',
+				},
+			},
+			required: ['userId', 'roomId'],
+			additionalProperties: false,
+		},
+		{
+			properties: {
+				username: {
+					type: 'string',
+				},
+				roomId: {
+					type: 'string',
+				},
+			},
+			required: ['username', 'roomId'],
+			additionalProperties: false,
+		},
+	],
+};
+
+export const isRoomsIsUserMuted = ajv.compile<RoomsIsUserMuted>(RoomsIsUserMutedSchema);
```

Comment:
```suggestion
export const isRoomsIsUserMutedProps = ajv.compile<RoomsIsUserMuted>(RoomsIsUserMutedSchema);
```
Just a small detail, but I think that's the naming pattern.